jupyterhub/services/auth.py

```diff
@@ -832,8 +833,12 @@ def get_login_url(self):
             # add state argument to OAuth url
             state = self.hub_auth.set_state_cookie(self, next_url=self.request.uri)
             login_url = url_concat(login_url, {'state': state})
-        app_log.debug("Redirecting to login url: %s", login_url)
-        return login_url
+        # override at setting level,
```

Comment:
```suggestion
        # temporary override at setting level,
```

Just an adjective to help those not previously familiar with https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict